packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -238,31 +238,27 @@ export const VtexCommerce = (
     },
     session: (search: string): Promise<Session> => {
       const params = new URLSearchParams(search)
+      const authCookie = {
+        name: `VtexIdclientAutCookie_${account}`,
+        value: getCookie(
+          `VtexIdclientAutCookie_${account}` ?? null,
```

Comment:
I'm confused about this `?? null` it looks like it's in the incorrect place? Should this be a default value for the "value" or for the first argument of the `getCookie` call?

Reply:
Sure! My bad :) Adjusting here now